src/components/shared/landscape-view-manager/landscape-view-manager-component.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
+
+const LandscapeViewManager = ({ view, zoomLevelTrigger, onZoomChange, query, isLandscapeMode }) => {
+  const [ isInteracting, setInteracting ] = useState(false);
+  const [ isUpdating, setUpdating ] = useState(false);
+  const [ landscapeView, setLandscapeView ] = useState(false);
+
+  const isLandscapeViewOnEvent = (zoomValue, landscapeView) => zoomValue >= zoomLevelTrigger && landscapeView === false;
```

Comment:
can be simplified to `zoomValue >= zoomLevelTrigger && !landscapeView`

Reply:
👍